src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../services/users';
+import { User } from '../../types/User';
+
+type Props = {
+  setHasTodoClick: (a: boolean) => void;
+  fillteredTodos: Todo[];
```

Comment:
The variable 'fillteredTodos' is misspelled. It should be 'filteredTodos' to correctly reflect its purpose of storing filtered todo items.